src/components/VolunteerDashboard/UnsubmittedLogDraft.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Flex, Box, Button, Text } from '@chakra-ui/react';
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'Box' is defined but never used.